tools/release.sh

```diff
@@ -13,6 +13,10 @@ declare -A languages=(
   ["es"]="Espanol"
   ["fr"]="Francais"
   ["ua"]="Ukrainska"
+  ["cs"]="Cestina"
+  ["ru"]="Russkiy"
+  ["he"]="Ivrrit"
```

Comment:
@davidpodless is this correct romanization of Hebrew?

Reply:
Hi, sorry I didnt answer before

Ivrit is more common, but it is understandable regardless, so I don't mind it